common/src/main/java/com/microsoft/identity/common/internal/result/MsalBrokerResultAdapter.java

```diff
@@ -610,8 +613,60 @@ public Bundle extractInteractiveRequestBundleFromResultBundle(@NonNull final Bun
         return resultBundle;
     }
 
+    /**
+     * Get authorizationResult from resultBundle for Device Code Flow
+     * @param resultBundle The bundle to interpret
+     * @return authorizationResult {@link AuthorizationResult}
+     * @throws BaseException
+     * @throws ClientException
+     */
+    @NonNull
+    public AuthorizationResult getDeviceCodeFlowAuthResultFromResultBundle(@NonNull final Bundle resultBundle) throws BaseException, ClientException {
+        String serializedDCFAuthResult = resultBundle.getString(AuthenticationConstants.Broker.BROKER_DCF_AUTH_RESULT);
```

Comment:
>serializedDCFAuthResult [](http://example.com/codeflow?start=15&length=23)

Why this has not been part of BrokerResult?

Reply:
one reason i can think of that this is not part of generic broker result, we have this only for the case of DCF flow and make sense to add it as part of bundle. Similar pattern is followed for other result like "BROKER_GENERATE_SHR_RESULT" in same file.

Reply:
please let me know if you thing otherwise, for now resolving this comment.